# HG changeset patch
# User John Donoghue <john.donoghue@ieee.org>
# Date 1406465631 14400
#      Sun Jul 27 08:53:51 2014 -0400
# Branch gui-release
# Node ID 3ed674758e0d6321de82e46402e104771345c05e
# Parent  63c5f95afeb3a9e83e53d37c001be82d5b0dc13a
Disable bell in Windows (Bug #42268)

* scripts/startup/inputrc: if term=cygwin, set bell style to none

diff --git a/scripts/startup/inputrc b/scripts/startup/inputrc
--- a/scripts/startup/inputrc
+++ b/scripts/startup/inputrc
@@ -19,8 +19,13 @@
 ##   sequence for the DOWN arrow.
 
 "\e[B": history-search-forward
 
 ## Disable so the usual paste shortcut will work on Windows systems.
 ## \C-q should still be available for quoted insert.
 
 "\C-v": ""
+
+## In windows, disable audiable bell which is enabled by default
+$if term=cygwin
+   set bell-style none
+$endif
